WordPress/openverse
# Rename catalog requirements files to match Renovate pattern

## Description



I recently noticed that [Airflow v2.7.3 was released last month](https://airflow.apache.org/docs/apache-airflow/stable/release_notes.html#airflow-2-7-3-2023-11-04), but we hadn't received a Renovate PR to update it. They were added in #3465 but should have been updated since then.

I added Renovate debug logging using the following and ran the Renovate workflow on that branch:
```diff
diff --git a/.github/workflows/renovate.yml b/.github/workflows/renovate.yml
index bd98989f6..4543bcd5f 100644
--- a/.github/workflows/renovate.yml
+++ b/.github/workflows/renovate.yml
@@ -68,6 +68,7 @@ jobs:
         env:
           # This enables the cache -- if this is set, it's not necessary to add it to renovate.json.
           RENOVATE_REPOSITORY_CACHE: ${{ github.event.inputs.repoCache || 'enabled' }}
+          LOG_LEVEL: "debug"
 
       # Compression helps performance in the upload step!
       - name: Compress renovate cache
```

While looking through the lines, I saw this:

```
2023-12-12T00:31:34.1520637Z DEBUG: Using file match: (^|/)[\w-]*requirements(-\w+)?\.(txt|pip)$ for manager pip_requirements (repository=WordPress/openverse)
...
2023-12-12T00:31:34.7391009Z DEBUG: Matched 1 file(s) for manager pip_requirements: archive/requirements.txt (repository=WordPress/openverse)
```

Note that the pattern Renovate is expecting is `requirements-` not `requirements_`! Additionally, the catalog requirements are missing from the `pip_requirements` that Renovate is looking for. So I think the underscore was preventing Renovate from finding the catalog's requirements.

This PR changes the files to match the appropriate, expected pattern.

## Testing Instructions


I _think_ we'll have to merge this and then run the Renovate workflow and see if it helps!


## Checklist



[best_practices]:
  https://git-scm.com/book/en/v2/Distributed-Git-Contributing-to-a-Project#_commit_guidelines

## Developer Certificate of Origin


<details>
<summary>Developer Certificate of Origin</summary>

```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

</details>
